### DISCUSSION TITLE
Undefined error in Swissup_Breeze/js/core/define.js

### DISCUSSION THREAD

**Discussion by rutulritz** [breezefront/community#95] [1. Help]
Hi,

I am getting below error on every pages on my website. 

Can you please help me why this error is showing in console?

You can check it here : https://colehaan.co.id/

![image](https://user-images.githubusercontent.com/9527439/231385826-c0b4eef6-3255-4e56-a268-85b1773a6231.png)

Thanks

**Comment by vovayatsyuk** [comment]
Fixed: https://github.com/breezefront/module-breeze/commit/b0363343a542824c696e0e60f973af19588bd50e

As a workaround, you can change your code as follows (add empty callback as a second argument):

```js
require(['https://www.google.com/recaptcha/api.js?onload=onloadCallbackGoogleRecapcha&render=explicit'], () => {});
```

**Reply by rutulritz** [reply]
Thanks you @vovayatsyuk . now it is fixed.
